balarco/utils.py

```diff
@@ -16,118 +17,31 @@
 GROUP_SUPERUSUARIO = "Super usuario"
 
 
-def generic_rest_list_objects(request, serializer_class, obj_class):
-    """Function to list all objects of a model in GenericViewSet:list.
-    It makes a Query to the specified object class geting all active objects
-
-    Parameters
-    ----------
-    request: request
-        The request that was made by the client
-    serializer_class: class
-        Class of the model serializer
-    obj_class: class
-        Class of the model
-
-    Returns
-    -------
-    Response
-        Response object containing the serializer data
-    """
-    queryset = obj_class.objects.filter(is_active=True)
-    serializer = serializer_class(queryset, many=True)
-    return Response(serializer.data, status.HTTP_200_OK)
+def save_object_from_data(obj_class, serializer_class, data):
+    obj_serializer = serializer_class(data=data)
+    if obj_serializer.is_valid():
+        obj_class.objects.create(**obj_serializer.validated_data)
+        return True
+    else:
+        return False
```

Comment:
Not obligatory because of time. But we should consider raising exceptions if not valid